src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1138,13 +1138,13 @@ static LinkAnimationHeader* D_808543D4[] = {
     &gPlayerAnim_002CC0,
 };
 
-// return type can't be void due to regalloc in func_8084FCAC
+// return type must be s32 due to regalloc in func_8084FCAC
```

Comment:
we dont know it has to be s32 specifically, so i think the original wording was fine

Reply:
I had changed it to f32 to try a different return and it still didn't match (even when returning nothing)

Reply:
any integer type should be exchangeable here. float is a special case cause it wouldnt necessarily use v0